lib/container.ex

```diff
@@ -67,17 +71,29 @@ defmodule RemoteDockers.Container do
   def list_all!(_), do: raise ArgumentError.exception("Invalid host config type")
 
   @doc """
-  Create a container
+  Create a container from the specified container's image configuration, or a simple image name.
+
+  ## Examples:
+  ```elixir
+  image_config =
+    ImageConfig.new("MyImage")
+    |> ImageConfig.add_env("MY_ENV_VAR", "my_env_var_value")
+  Container.create!(HostConfig.new(), "my_container", image_config)
+  ```
```

Comment:
You can use
```elixir
  ## Examples

      iex> MyApp.Hello.world(:john)
      :ok
```

It will be executed by
```elixir
doctest RemoteDockers.Container
```

Reply:
That's a good idea, but it cannot be used for such connectors, since a specific host configuration is needed to run these tests..!

Reply:
Yes I have test it before... But in this case, it can be relevant ;-)